x-pack/plugins/actions/server/sub_action_framework/register.test.ts

```diff
@@ -55,6 +56,7 @@ describe('Registration', () => {
       supportedFeatureIds: connector.supportedFeatureIds,
       validate: expect.anything(),
       executor: expect.anything(),
+      renderParameterTemplates: expect.anything(),
```

Comment:
nit: This check does not asserts that the correct function is passed. What about 

```
const renderParameterTemplates = jest.fn()

const connector = {
    id: '.test',
    name: 'Test',
    minimumLicenseRequired: 'basic' as const,
    supportedFeatureIds: ['alerting'],
    schema: {
      config: TestConfigSchema,
      secrets: TestSecretsSchema,
    },
    Service: TestSubActionConnector,
   renderParameterTemplatesm,
  };

// code omitted for brevity

 expect(actionTypeRegistry.register).toHaveBeenCalledWith({
      id: connector.id,
      name: connector.name,
      minimumLicenseRequired: connector.minimumLicenseRequired,
      supportedFeatureIds: connector.supportedFeatureIds,
      validate: expect.anything(),
      executor: expect.anything(),
      renderParameterTemplates,
    }); 

```

Reply:
Sure, I was following the same criteria as in the `validate` and `executor` properties. 
However, strictly speaking, it does not need to be the same function, the important thing is the function is properly wired.
Instead of testing the function reference (it could change and still be correct), I updated the test by adding a test case that checks the `renderParameterTemplates` is called with the correct parameters and it returns the correct value.

Reply:
You are right @semd! We should improve that. Thanks for the additional test. Much better.

Reply:
@cnasikas 👍 thanks for the review!